README.md

```diff
@@ -69,7 +69,7 @@ Otherwise set the `spark.neo4j.bolt.url` in your `SparkConf` pointing e.g. to `b
 You can provide user and password as part of the URL `bolt://neo4j:<password>@localhost` or individually in `spark.neo4j.bolt.user` and `spark.neo4j.bolt.password`.
 
 If you're running Neo4j with the Bolt connector and the option `dbms.connector.bolt.tls_level` in Neo4j is `REQUIRED`, you must set the `spark.neo4j.bolt.encryption.status` to `true` in your `SparkConf`.
-Otherwise, you can either ignore `spark.neo4j.bolt.encryption.status` or  set  `spark.neo4j.bolt.encryption.status` to `false` in your `SparkConf`.
+Otherwise, you can either ignore `spark.neo4j.bolt.encryption` or  set  `spark.neo4j.bolt.encryption` to `false` in your `SparkConf`.
 
 ## Builder API
```

Comment:
can you make sure in the readme the other version numbers that are mentioned are fixed / correct?

Reply:
Done